### PR TITLE
Update Team1x1.mjs

### LINKED ISSUE #2744: [team1x1] Mangas redirecting to new domain
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
No

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
http://team1x1.com Changed to https://tqneplus.com

**To Reproduce**
1. Name of connector team1x1.com

### PR DESCRIPTION
**Pull request by Robonau**
new url 
get chapters pages

these guy seem to be changing the url frequently, there doesn't seem to be a pattern though so i don't know if we can do anything about it  
resolves #2744

### CODE REVIEW

File: src/web/mjs/connectors/Team1x1.mjs [hunk 1]
```diff
@@ -7,56 +7,69 @@ export default class Team1x1 extends Connector {
...
+        const data = await this.fetchDOM(request, 'div.pagination div.wp-pagenavi > a.last');
+        const pageCount = parseInt(data[0].href.split("/").slice(-2, -1)[0].trim());
+        const pageLinks = [... new Array(pageCount).keys()].map(page => request.url + 'page/' + (page + 1) + '/');
+        return await this._getMangaListFromPages(pageLinks);
+    }
+
+    async _getChapters(manga) {
+        let chapterList = [];
+        const request = new Request(this.url + manga.id, this.requestOptions);
+        const data = await this.fetchDOM(request, 'span.current');
```
Comment (27136773) [inline src/web/mjs/connectors/Team1x1.mjs]: May improve query to make it more robust `div.pagination span.current`.

Is `span.current` really the last page? Sounds to me more like it is the current page (which is probably always `1`).

Comment (27136773) [reply src/web/mjs/connectors/Team1x1.mjs]: OK, seems the website automatically sets the last page as the current, e.g.
https://tqneplus.com/manga/the-heavens-list/

image

**Review by 27136773 — APPROVED** [review]
Looks good

File: src/web/mjs/connectors/Team1x1.mjs (hunk 1 above)
Comment (Robonau) [reply src/web/mjs/connectors/Team1x1.mjs]: yea, as u said it is the current page and that is the last page, i think it's an arabic right to left thing

**Pull request merged**